tests/admin_inlines/tests.py

```diff
@@ -1306,6 +1307,7 @@ def test_inline_formset_error_input_border(self):
                 '1px solid #ba2121',
             )
 
+    # test will fail
```

Comment:
Does the test need fixing to work with the new behavior? Or is this change breaking expected behavior?

Reply:
@pope1ni  @carltongibson  The reason is that as I mentioned in  [ticket-9107](https://code.djangoproject.com/ticket/9107), by default it will show 3 forms in the form set or the number of forms that are mentioned in the extra property in the inline model. 

In the test, the query parameter is trying to reference the 10th form in the inline formset which doesn't exist as only 3 forms will be shown, so here I have a dilemma, should I override the extra parameter of the inline model based on the query parameter or keep everything as it is?

One suggestion could be to have a parameter in the query to override how many forms for the inline formset to show, but that could make a simple feature more complex. 

@pope1ni  @carltongibson  any ideas on this?